src/main/java/edu/hm/hafner/analysis/parser/CMakeParser.java

```diff
@@ -0,0 +1,45 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.RegexpDocumentParser;
+
+/**
+ * A parser for CMake warnings.
+ *
+ * @author Uwe Brandt
+ */
+public class CMakeParser extends RegexpDocumentParser {
```

Comment:
`RegexpDocumentParser` is deprecated, please use the `RegexpLineParser` or `LookaheadParser` as base class. Examples:
- https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/MavenConsoleParser.java
- https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/parser/Gcc4CompilerParser.java

You need to split your regexp: it should match only the single line. Then you can consume additional lines using `lookahead`.